### PR TITLE
Add discussion of how student editors are involved in the review process

### PR DESCRIPTION
**Pull request by davidlmobley**
I totally forgot to include any mention of student editors in the instructions for editors. This attempts to add some.

It will need checking, as I am not TOTALLY sure what we intend or what we can do via Scholastica. Specifically, as I've written it, each manuscript will involve a Student Editor who will either (a) review the manuscript, or (b) find someone else who will review the manuscript. This basically means they'd function sort of like an editor, managing review by a student (who might be them or might be someone else).

However, I am not sure if (i) that's what we want, and (ii) Scholastica will allow that, since it'd essentially mean we'd have two people functioning as editors on the same document and soliciting reviews. 

Alternatively (and perhaps this is what we intended) the Student Editors we have will BE our only student reviewers, and if they get overloaded we just add more.

@dmzuckerman @mrshirts @hmayes , thoughts?

### CODE REVIEW

**Comment by mrshirts** [comment]
> Alternatively (and perhaps this is what we intended) the Student Editors we have will BE our only student reviewers, and if they get overloaded we just add more.

I'm not sure this works, because then people know who some editors will be, which we definitely don't want for people early in their career. Too soon to make enemies?

So I think that there are actually 2 ways to involve non-PI level people here.

1. Make it more clear that we will have student reviewers.  There are 3 ways I can think of getting people. 
    i. Mention somewhere (in author instructions?) to nominate student reviewers so we have a list of potential reviewers to turn to. 
    ii. Editors can generate a set of grad students/ postdoc they have run into. 
    iii. The invitation letter for reviewing could also ask the reviewer if they have a graduate student or postdoc they would like to suggest to review, instead of, or in addition to, their own (though we have to make sure that we don't get everybody just suggesting their graduate students).

2. Separately, we create a new category, such as "Editorial Board Assistants", or something like that, for students and postdocs that want to help, so we can give them some credit.  We would have to add to the bylaws, but the definition would be something relatively simple such as "May assist with other tasks as required, such as preparing the website", but exclude them from actually serving as associate editor or section lead editor.  Other thoughts: they could participate in discussions (so they get experience), but not vote?

**Comment by davidlmobley** [comment]
Right now, our bylaws don't discuss the category of Student Editor at all, I don't think. 

I think my preferred solution is just to make it more clear we will have student reviews, without specifying where they will come from, and our Student Editors can in some cases serve as student reviewers, but also can assist in developing a network of student reviewers. We would probably want to make clear in our bylaws that this is their role -- helping ensure accessibility, and perhaps providing input on editorial decisions from a student/postdoc point of view, but not voting.

I don't think we want to solicit student reviewers in our reviewer invitation letter necessarily; I worry that doing so would result in too many "PI AND a student in the group review the paper" scenarios which seems like overrepresentation of one group, and would also sort of dissuade PIs from doing it themselves since they could just ask a student to instead.

**Comment by dmzuckerman** [comment]
Let me try to synthesize some of the discussion and make some further suggestions.  

There are some principles:
- As @mrshirts said, we want to take special care to preserve the anonymity of student reviewers.
- I also think we should avoid creating a situation where we put pressure on a student to take on a position/commitment, which implicitly puts a burden on the student's PI.  (Just by creating the 'student editor' position, we might create some osmotic pressure.)
- We should welcome and reward above-and-beyond contributions by anyone at the non-PI level: students, postdocs, staff scientists, ...
- As @davidlmobley said, we should develop a network of student reviewers.  Equally importantly, we should develop infrastructure for such a network.
- I do not think we should view graduate students as supplying labor (e.g., for website management).  That seems too non-academic to me and could end up harming the student in the long run by distracting from research and publication.

Based on the above, my suggestions are:
- Do not require participation by a 'student editor' for each submission ... just a student reviewer.
- Task the associate and section lead editors with ensuring student anonymity, if necessary by editing the student review.  I don't like the idea of having student names suggested because of the heightened possibility of bias or non-anonymity.  We should be in charge of find the students (see below).
- We can consider creating nonvoting positions called 'Advisory Editors' which would not require manuscript 'management' (i.e., editorial work), but would be a mechanism to reward high commitment and also provide a formal conduit for input to the editorial board by folks besides the board.  Such advisory editors might come from any level - student, postdoc, etc.   (We could consider having at least the possibility of 'Senior Advisory Editors' to create a mechanism for external feedback from some of the more senior members of our community.)
- Our website should feature a 'For students' or 'For junior scientists' tab that should advertise the roles available to non-PIs.  A key component could be a google form or similar via which students could input their contact info and expertise/interests.  We and everyone we know should steer students to this.
- I think we should hire a consultant to handle the website stuff for us.  We can easily find someone for ~$100/hour and I don't think it should be too many hours.  If it does turn out to be expensive, then surely it's too much for a student.

**Comment by davidlmobley** [comment]
I agree with all of Dan's points here. @mrshirts do you agree?

If so someone should make changes to this PR and to our bylaws document accordingly. I could address this one if I do it right now, but otherwise won't have time to do it today.

Please "assign" this to me if you want me to do it. :)

**Comment by mrshirts** [comment]
If you can do it today, that would be great. I apparently can't work nearly as fast as you.  I can look at it afterwards if you can't.

**Comment by davidlmobley** [comment]
Not gonna get to it soon, it doesn't look like. Assign me to it if you need me to work on it, but it probably wouldn't be til next week unless things change.

**Comment by davidlmobley** [comment]
My perspective would be to not complicate things right now, and so to only have student reviews, and not add any additional categories of editors, etc. I'd rather keep things as simple as possible until we get some momentum build up, then make it more complex, rather than adding a bunch of layers of complexity at the beginning and making it harder to build momentum. And likewise for website stuff for students, plus I'm with you in that I'd rather pay someone to do it (it doesn't really yield academic credit, and our students aren't usually trying to become web designers).

Bylaws: They are in Michael's `bylaws` branch (e.g. in this PR: https://github.com/livecomsjournal/livecomsjournal.github.io/pull/86) so you want to `git fetch` then checkout bylaws and then make a new branch off of it (if you want to make changes there, that is). Then you do a PR from your new branch to his PR.

**Comment by mrshirts** [comment]
> My perspective would be to not complicate things right now, and so to only have student reviews, and not add any additional categories of editors, etc. I'd rather keep things as simple as possible until we get some momentum build up

Agreed.

> And likewise for website stuff for students, plus I'm with you in that I'd rather pay someone to do it (it doesn't really yield academic credit, and our students aren't usually trying to become web designers).

Agreed. Key will be to find someone who can do it, though.  Do we have any leads?   I think the current design mod tweaks should be fine for now.

File: _editorial_policies/02_editor_instructions.md [hunk 1]
```diff
@@ -30,7 +30,13 @@ The Lead Editor:
...
-- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two reviewers, though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.
+- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two reviewers as well as a student reviewer (a Student Editor or a student reviewer recruited by a Student Editor), though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.
+
+### Student Editors
```
Comment (mrshirts) [inline _editorial_policies/02_editor_instructions.md]: Leave this out.

**Comment by dmzuckerman** [comment]
I suggest: those of you who know what expertise is required to maintain website (@mrshirts @davidlmobley @ptmerz ?) please sketch a 'job description' we can circulate to consultants we know or post to a job board.   I have a guy who maintains a Drupal site for me who might work, but I want to be able to give him specifics.

**Comment by mrshirts** [comment]
my summary (and will do a PR to this PR this morning to reflect)

* we nix student editors as a position.

* We hold off on "advisory editors" for now.

* We create a "for early career scientists" page, where we talk about opportunities for students/postdocs/etc. 

* We create a thanks page that thanks specific people/organizations who have helped who are not on the editorial board (Pascal, Justin, Merry so far, CU Boulder libraries).  

* We look for a web maintainer.

**Comment by davidlmobley** [comment]
Sounds good, @mrshirts .

**Comment by hmayes** [comment]
Agreed!

**Pull request merged**